docs/eventing/flows/sequence.md

```diff
@@ -57,14 +57,14 @@ through each stage.
 ### [Sequence with no reply (terminal last Step)](../samples/sequence/sequence-terminal/README.md)
 
 For the first example, we'll use a 3 Step `Sequence` that is wired directly into
-the `CronJobSource`. Each of the steps simply tacks on "- Handled by
+the `PingSource`. Each of the steps simply tacks on "- Handled by
```

Comment:
Any PingSource yamls that need to be updated with the proper kind or apiVersion?

Reply:
They all have been already ported, for example:
https://github.com/knative/docs/tree/master/docs/eventing/samples/sequence/sequence-terminal

And:
https://github.com/knative/docs/blob/master/docs/eventing/samples/sequence/sequence-with-broker-trigger/ping-source.yaml